src/verifai/samplers/scenic_sampler.py

```diff
@@ -210,9 +212,9 @@ def __init__(self, scenario, maxIterations=None, ignoredProperties=None):
         super().__init__(space)
 
     @classmethod
-    def fromScenario(cls, path, maxIterations=None, ignoredProperties=None):
+    def fromScenario(cls, path, maxIterations=None, ignoredProperties=None, model=None, **params):
```

Comment:
I would not have `params` be a set of keyword arguments here: we should separate arguments that `fromScenario` needs from global parameters that should be passed through to the scenario.